### ISSUE TITLE
Credits Used Value in Serverless Framework Dashboard Remains Unchanged

### ISSUE THREAD

**Issue by Hiroki-Aoki** [serverless/serverless#12817]
### Issue description

I have 15 Credits. And I have deployed seven CloudFormation stacks using a license key with Serverless Framework V.4. Expectedly, the Credits Used should have increased by 7, but the value in the Serverless Framework Dashboard remains at 0. At first, I thought, "Maybe it's because less than five days have passed?", but even though five days have passed since I created the CloudFormation stacks, the credits used have not changed. Is this a bug in the Serverless Framework Dashboard? If so, is there a plan to fix it?
 
Version used:
Serverless Framework: 4.3.2

**Comment by eahefnawy** [comment]
Hi @Hiroki-Aoki thank you for reporting. By "CloudFormation stacks" do you mean that you do not use `serverless.yml`, but rather deploying raw CloudFormation stacks [as described here](https://www.serverless.com/framework/docs/guides/sam)?

**Comment by garethmcc** [comment]
@Hiroki-Aoki I would also like to ask if you could get in touch via support using support@serverless.com as there may be questions we need to ask with account information we would not want to share on a public forum to assist further with the issue you describe.

There are potentially a number of reasons for what you are seeing and we would need to look into your account to check into that.

**Comment by Hiroki-Aoki** [comment]
> Hi @Hiroki-Aoki thank you for reporting. By "CloudFormation stacks" do you mean that you do not use `serverless.yml`, but rather deploying raw CloudFormation stacks [as described here](https://www.serverless.com/framework/docs/guides/sam)?

Thank you for your quick reply. I'm having our verification member check the fact, so please wait a moment.

**Comment by Hiroki-Aoki** [comment]
> @Hiroki-Aoki I would also like to ask if you could get in touch via support using [support@serverless.com](mailto:support@serverless.com) as there may be questions we need to ask with account information we would not want to share on a public forum to assist further with the issue you describe.
> 
> There are potentially a number of reasons for what you are seeing and we would need to look into your account to check into that.

Thank you for your quick reply. I've read your email regarding this question and will respond shortly.

**Comment by Hiroki-Aoki** [comment]
@garethmcc I've replied to your email, so please check it.

**Comment by Hiroki-Aoki** [comment]
> Hi @Hiroki-Aoki thank you for reporting. By "CloudFormation stacks" do you mean that you do not use `serverless.yml`, but rather deploying raw CloudFormation stacks [as described here](https://www.serverless.com/framework/docs/guides/sam)?

@eahefnawy No. The CloudFormation stacks was created using serverless.yml.
